tests/utils/query_builder_test.py

```diff
@@ -41,7 +41,9 @@ def test_add_select_conditions_should_update_select_list(
     ],
 )
 def test_add_sql_in_condition_should_update_where_list(
-    attribute, ids_list, expected_where_condition_list,
+    attribute,
```

Comment:
Is this re-formatting expected or the [pre-commit](https://ioetec.slack.com/archives/CUD8VFS86/p1588709542149600) action has not been set?

Reply:
It was the pre-commit action